packages/backend/migrations/20230201133042_create_ledger_accounts_table.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.createTable('ledgerAccounts', function (table) {
+    table.uuid('id').notNullable().primary()
+    table.uuid('accountRef').notNullable().unique().index()

```

Comment:
I want the `ids` of `ledgerAccounts` and `ledgerTransfers` to be used only internally, between these two tables. When creating an `incomingPayment` or `outgoingPayment`s, the `incoming/outgoingPayment.ids` will be actually the `accountRef`s instead.